packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-12/actions.test.js

```diff
@@ -0,0 +1,68 @@
+import test from 'ava';
+
+import { agd, agoric, agops } from '../cliHelper.js';
+import { GOV1ADDR, SDK_ROOT } from '../constants.js';
+import { installBundles, runZcfUpgrade, runProber } from './actions.js';
+import { adjustVault, closeVault, mintIST, openVault } from '../econHelpers.js';
+
+test.before(async t => {
+  await mintIST(GOV1ADDR, 12340000000, 10000, 2000);
+
+  const bundlesData = [
+    {
+      name: 'Zcf-upgrade',
+      filePath: `${SDK_ROOT}/packages/zoe/src/contractFacet/vatRoot.js`,
+    },
+    {
+      name: 'Zoe-upgrade',
+      filePath: `${SDK_ROOT}/packages/vats/src/vat-zoe.js`,
+    },
+    {
+      name: 'prober-contract',
+      filePath: `${SDK_ROOT}/packages/boot/test/bootstrapTests/zcfProbe.js`,
+    },
+  ];
+
+  // @ts-expect-error
+  t.context.bundleIds = await installBundles(bundlesData);
+});
+
+test.skip('Open Vaults', async t => {
```

Comment:
Why are all the tests in this file and packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-12/post.test.js skipped?